packages/cactus-plugin-satp-hermes/src/main/typescript/plugin-satp-hermes-gateway.ts

```diff
@@ -181,6 +192,25 @@ export class SATPGateway implements IPluginWebService, ICactusPlugin {
     this.OAPIServerEnabled = this.config.enableOpenAPI ?? true;
 
     this.OAS = OAS;
+
+    if (this.config.enableCrashManager) {
+      const crashOptions: ICrashRecoveryManagerOptions = {
+        instanceId: this.instanceId,
+        logLevel: this.config.logLevel,
+        bridgeConfig: this.bridgesManager,
+        orchestrator: this.gatewayOrchestrator,
+        localRepository: this.localRepository,
+        remoteRepository: this.remoteRepository,
+        signer: this.signer,
+        pubKey: this.pubKey,
+      };
+      this.crashManager = new CrashManager(crashOptions);
+    }
+
+    if (this.config.enableMigration) {
```

Comment:
is this about enalbing the database connection? if so, please change the variable names and